### ISSUE TITLE
Cannot navigate collections dropdown via keyboard in "New Collection" dialog on windows

### ISSUE THREAD

**Issue by abaevbog** [zotero/zotero#5202]
https://forums.zotero.org/discussion/123329/accessibility-creating-new-collections-keyboard-problem

This does not occur on a mac. On windows, once the collections dropdown is focused, space does not open it. Strangely, arrow up/down can change the selected collection once or twice (without opening the popup), after which they also stop having any effect.

**Comment by abaevbog** [comment]
Looks like on windows, space does not open a `<menulist>`  in general but Alt+ArrowDown does. It works to open the collections menu but arrows do not navigate the menu.

**Comment by abaevbog** [comment]
You see the menu jumping behind the dialog too? Great! I wasn't sure if it would end up being related to keyboard nav or not, so I added a separate #5204 for it

**Comment by abaevbog** [comment]
I'll also note that navigating `menulist`s on windows in general feels quite off to me. If the menu does open (e.g. in preferences, or item type dropdown in `itemBox`), attemting to navigate menuitems with arrowUp/Down immediately changes the selection of the `menulist`, which then closes the menu. So I seemingly can't move farther than 1 menuitem from where I started before the menu hides.

For instance, to get to "Bill" item type, I would need to press ArrowUp twice. But as soon as I press ArrowUp once, this is what happens:

https://github.com/user-attachments/assets/47d3d3f7-4d73-4f27-a2db-ec937452c662
